actors/ethaccount/src/lib.rs

```diff
@@ -126,22 +126,14 @@ impl EthAccountActor {
 
         Ok(())
     }
-
-    // Always succeeds, accepting any transfers.
-    pub fn universal_receiver_hook(
-        rt: &mut impl Runtime,
-        _params: UniversalReceiverParams,
-    ) -> Result<(), ActorError> {
-        rt.validate_immediate_caller_accept_any()?;
-        Ok(())
-    }
 }
 
 impl ActorCode for EthAccountActor {
     type Methods = Method;
     actor_dispatch! {
         Constructor => constructor,
         AuthenticateMessageExported => authenticate_message,
-        UniversalReceiverHook => universal_receiver_hook,
+        UniversalReceiverHook => (),
```

Comment:
Will this skip decoding the ignore parameters too (#1066)? 🎉 

Could you revert [this change in the tests](https://github.com/filecoin-project/builtin-actors/pull/913/files#r1050211261) to demonstrate this?

Reply:
> Will this skip decoding the ignore parameters too (https://github.com/filecoin-project/builtin-actors/issues/1066)? 🎉

Yes.

> Could you revert [this change in the tests](https://github.com/filecoin-project/builtin-actors/pull/913/files#r1050211261) to demonstrate this?

Can do.